src/Features/Core/Portable/EditAndContinue/EditAndContinueService.cs

```diff
@@ -18,14 +24,49 @@ namespace Microsoft.CodeAnalysis.EditAndContinue
     /// </remarks>
     internal class EditAndContinueService : IEditAndContinueService
     {
+        private readonly IActiveStatementProvider _activeStatementProvider;
         private readonly IDiagnosticAnalyzerService _diagnosticService;
         private DebuggingSession _debuggingSession;
         private EditSession _editSession;
 
-        public EditAndContinueService(IDiagnosticAnalyzerService diagnosticService)
+        // TODO:
```

Comment:
>TODO: [](start = 11, length = 5)

any work item associated with this TODO?

Reply:
No, just update the comment.

---
In reply to: [162476978](https://github.com/dotnet/roslyn/pull/24316#discussion_r162476978) [](ancestors = 162476978)